app/views/projects/_settings_sidebar.html.erb

```diff
@@ -0,0 +1,24 @@
+<li class="sidebar-item sidebar-brand">
+  PROJECT SETTINGS
+</li>
+
+<% if policy(@project).manage? %>
```

Comment:
Avoid using instance variables (`@project`) in partials, pass it as a parameter to the partial.